### PR TITLE
Added 8.3 Battle Pets

### PR DESCRIPTION
**Pull request by criggs11**

### CODE REVIEW

File: app/data/battlepets.json [hunk 1]
```diff
@@ -702,6 +702,48 @@
...
+        "id": "dbddd3a4",
+        "items": [
+          {
+            "creatureId": 162499,
+            "icon": "inv_icon_wing03d",
+            "name": "Oasis Void-Duster",
+            "spellid": null
+          },
+          {
+            "creatureId": 154796,
```
Comment (seirl) [inline app/data/battlepets.json]: This is the creature ID of the Clanking Scrapsorter, not the Void-Scarred Locust.

File: app/data/battlepets.json [hunk 2]
```diff
@@ -702,6 +702,48 @@
...
+        "id": "0d5fc9cd",
+        "items": [
+          {
+            "creatureId": 162525,
+            "icon": "inv_pet_cricket",
+            "name": "Golden Chirper",
+            "spellid": null
+          },
+          {
+            "creatureId": 162502,
```
Comment (seirl) [inline app/data/battlepets.json]: And this is the ID of the Void-Scarred Locust, not the Voidglower.

File: app/data/battlepets.json (hunk 1 above)
Comment (criggs11) [reply app/data/battlepets.json]: Fixed.

File: app/data/battlepets.json (hunk 2 above)
Comment (criggs11) [reply app/data/battlepets.json]: Fixed.

**Comment by seirl** [comment]
Thanks!

**Pull request merged**